media_softlet/agnostic/common/codec/hal/enc/hevc/features/encode_hevc_basic_feature_rsvd.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+* Copyright (c) 2021, Intel Corporation
```

Comment:
Please modify copyright 2021 -> 2023. And same for other new files.

Reply:
1. Yes, ill delete the embargo rsvd files.
2. Yes, ill rename the file.
Thanks for you reminder.